pachyderm/values.yaml

```diff
@@ -92,13 +91,11 @@ etcd:
     tag: "v3.3.5"
     pullPolicy: IfNotPresent
   storageSize: "10Gi"
-  resources: {}
-    #limits:
-    #  cpu: "1"
-    #  memory: 2G
-    #requests:
-    #  cpu: "1"
-    #  memory: 2G    
+  cpuRequest: 1
```

Comment:
We should check to make sure this still works, I'm not sure you can partially specify a resources spec? I've mostly seen helm charts do it the above commented out way, but am open to changing.

Reply:
Updated to take a full resources everywhere.